hack/prepare-for-integration-tests.sh

```diff
@@ -90,6 +91,15 @@ while (("$#")); do
     get_active_directory_vars=$1
     shift
     ;;
+  --alternate-deploy)
```

Comment:
Could we add a new line to the `--help` output to describe this new `--alternate-deploy` flag?